graphbolt/include/graphbolt/utils.h

```diff
@@ -0,0 +1,76 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file graphbolt/utils.h
+ * @brief Header file of utils in graphbolt.
+ */
+#ifndef GRAPHBOLT_UTILS_H_
+#define GRAPHBOLT_UTILS_H_
+
+#include <omp.h>
+
+namespace graphbolt {
+namespace utils {
+
+using RangePickFn = std::function<torch::Tensor(
+    int64_t start, int64_t end, int64_t num_samples)>;
+
+inline size_t compute_num_threads(size_t begin, size_t end, size_t grain_size) {
```

Comment:
And create a threading_utils.h for this method, under graphbolt namespace.

Reply:
Make sense, use `runtime` as namespace for more usage.